### PR TITLE
Add extra employee field

### PR DESCRIPTION
**Pull request by wobinb**
## Description
New field TerminationReason has been added at API level

## Release Notes
Full detail on the field here:
https://developer.xero.com/documentation/api/payrollau/stp-changes/#termination-reason

### CODE REVIEW

**Comment by wobinb** [comment]
I've added this an an ENUM, would it be better not an ENUM?

**Comment by wobinb** [comment]
This has been requested by a user of the PHP SDK

**Comment by SerKnight** [comment]
Going to see if there is a way to add metadata to an enum. If these are to ever fuel docs, just looking at those values doesn't really help me understand the options.. Maybe we have to put the full reasons in the comment?

```
enum:
          - V
          - I
          - D
          - R
          - F
          - C
          - T
```

File: xero-payroll-au.yaml [hunk 1]
```diff
@@ -3317,6 +3317,18 @@ components:
           type: string
           x-is-msdate: true
           example: '/Date(1584662400000+0000)/'
+        TerminationReason:
+          description: The reason for termination
```
Comment (SerKnight) [inline xero-payroll-au.yaml:3321]:
```suggestion
```suggestion
          description: 
            * `V` Voluntary cessation - An employee resignation, retirement, domestic or pressing necessity or abandonment of employment
            * `I` Ill health - An employee resignation due to medical condition that prevents the continuation of employment, such as for illness, ill-health, medical unfitness or total permanent disability
            * `D` Deceased - The death of an employee
            * `R` Redundancy - An employer-initiated termination of employment due to a genuine redundancy or approved early retirement scheme
            * `F` Dismissal - An employer-initiated termination of employment due to dismissal, inability to perform the required work, misconduct or inefficiency
            * `C` Contract cessation - The natural conclusion of a limited employment relationship due to contract/engagement duration or task completion, seasonal work completion, or to cease casuals that are no longer required
            * `T` Transfer - The administrative arrangements performed to transfer employees across payroll systems, move them temporarily to another employer (machinery of government for public servants), transfer of business, move them to outsourcing arrangements or other such technical activities.
```
```

**Pull request merged**